proto/Cargo.toml

```diff
@@ -20,7 +20,8 @@ all-features = true
 prost = "0.7"
 prost-types = "0.7"
 bytes = "1.0"
-anomaly = "0.2"
+anyhow = {version = "1.0", default-features = false, git = "https://github.com/DaviRain-Su/anyhow.git", branch = "open-stderror"}
```

Comment:
The version of `anyhow` here gives a compile error:

```
error[E0433]: failed to resolve: use of undeclared crate or module `anyhow`
    |
272 | pub type BoxError = alloc::Box<dyn anyhow::StdError + Send + Sync>;
    |                                    ^^^^^^ use of undeclared crate or module `anyhow`

error: aborting due to previous error
```

Reply:
I have reset this change.